internal/gps/vcs_source.go

```diff
@@ -47,6 +47,10 @@ func (bs *baseVCSSource) getManifestAndLock(ctx context.Context, pr ProjectRoot,
 	}
 
 	m, l, err := an.DeriveManifestAndLock(bs.repo.LocalPath(), pr)
+	if os.IsNotExist(err) {
+		return prepManifest(nil), nil, nil
```

Comment:
This is essentially guaranteed to be unreachable by a slew of code above it - we shouldn't be able to make it this far if the dir doesn't exist. It should only be reachable under active sabotage (or, ofc, a bug) - e.g., some other process removing the dir within the source cache while a `dep` command is running.

Even under such circumstances, it's relatively unlikely _this_ would be the spot that happens to first notice the dir is absent. If it is, though, it's preferable to error out, rather than pass up the "zero" result, as non-error results get cached. If/when that cache becomes persistent across runs (#431), that incorrect information could plague future runs.

So, better to just propagate the error right back out than try to recover.